src/DependenciesPackager/Program.cs

```diff
@@ -184,7 +189,7 @@ static int Main(string[] args)
                 packagesVersion,
                 useCli,
                 quiet,
-                runtimes,
+                runtime,
```

Comment:
I'm aware of that, but it's easier to have a single parameter with the rids and just pass that, than break it into platform and architectures.